recovery-init

```diff
@@ -24,6 +24,76 @@
 ##############################################################################
 
 export PATH=/sbin:/bin:/usr/bin:/usr/sbin
+ANDROID_USB=/sys/class/android_usb/android0
+GADGET_DIR=/config/usb_gadget
+USB_FUNCTIONS=rndis
+LOCAL_IP=10.42.66.66
+
+
+write()
+{
+	echo -n "$2" > $1
+}
+
+# This sets up the USB with whatever USB_FUNCTIONS are set to via configfs
+usb_setup_configfs() {
+    G_USB_ISERIAL=$GADGET_DIR/g1/strings/0x409/serialnumber
+    mkdir $GADGET_DIR/g1
+    write $GADGET_DIR/g1/idVendor                   "0x2931"
+    write $GADGET_DIR/g1/idProduct                  "0x0A06"
+    mkdir $GADGET_DIR/g1/strings/0x409
+    write $GADGET_DIR/g1/strings/0x409/serialnumber "$1"
+    write $GADGET_DIR/g1/strings/0x409/manufacturer "Jolla Boat Loader"
```

Comment:
Ah sorry, missed this perviously... Still small change, could you leave manufacturer as Jolla